npm/packages/@socketsupply/socket-win32-x64/package.json

```diff
@@ -12,7 +12,11 @@
   ],
   "scripts": {
     "test": ":",
-    "install": "cd bin && node verify-platform.js win32 x64"
+    "preinstall": "node ./bin/install-pre-reqs.js && echo Pre requisite checks complete.\nUse\n> install-pre-reqs\nto run this again.",
+    "install": "cd bin && node verify-platform.js win32 x64 -"
+  },
+  "bin": {
```

Comment:
No need to add it to `bin`

Reply:
Hey @jwerle 
My thought is that ap devs can run
```sh
npx @socketsupply/socket-win32-x64
```

If they want to rerun the dependency installer.

Reply:
ah! I like this! Maybe we can just call the bin `socket-win32-x64-setup` and do the same thing for the others?

Reply:
@jwerle What would the npx call and bin look like exactly?

Reply:
```json
"bin": {
  "socket-win32-x64": "bin/socket-win32-x64-setup"
}
```

```sh
npx @socketsupply/socket-win32-x64 # executes the `socket-win32-x64` bin
```